scripts/scr_print/scr_print.gml

```diff
@@ -0,0 +1,11 @@
+/// Helper functoin function for printing debug messages with multiple args
+function print(){
```

Comment:
ah nice, `print` is much easier than `show_debug_message`

gamemakers way of handing arguments is pretty odd lol

Reply:
Yea, had me stuck for while! Wasnt sure why my int values werent showing in the console output 😅